azkaban-common/src/main/java/azkaban/project/JdbcProjectLoader.java

```diff
@@ -16,6 +16,18 @@
 
 package azkaban.project;
 
+import azkaban.database.AbstractJdbcLoader;
```

Comment:
I agree. But this code involves some hacks with the JDBC interfaces which I don't intend to push to master. That's the reason why I published this as [my fork](https://github.com/suvodeep-pyne/azkaban/tree/migrate-tool).